### PR TITLE
hardhatify BondingManager tests 👷🏼

### PR DESCRIPTION
**Pull request by kautukkundan**
What does this pull request do? Explain your changes. (required)
Hardhatify BondingManager tests 👷

- [x] setController
- [x] setUnbondingPeriod
- [x] setNumActiveTranscoders
- [x] setMaxEarningsClaimsRounds
- [x] transcoder
- [x] bond
- [x] unbond
- [x] rebond
- [x] rebondFromUnbonded
- [x] withdrawStake
- [x] withdrawFees
- [x] reward
- [x] updateTranscoderWithFees
- [x] slashTranscoder
- [x] claimEarnings
- [x] claimSnapshotEarnings
- [x] pendingStake
- [x] pendingFees
- [x] setCurrentRoundTotalActiveStake
- [x] transcoderStatus
- [x] isActiveTranscoder
- [x] delegatorStatus
- [x] isRegisteredTranscoder
- [x] isValidUnbondingLock
- [x] getTotalBonded

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        "printWidth": 140,
+        "tabWidth": 4,
+        "useTabs": false,
+        "singleQuote": false,
+        "bracketSpacing": true,
+        "explicitTypes": "always"
+      }
+    }
+  ]
+}
```
Comment (kyriediculous) [inline .prettierrc]:
```suggestion
{
  "arrowParens": "avoid",
  "bracketSpacing": true,
  "endOfLine":"auto",
  "printWidth": 120,
  "singleQuote": false,
  "tabWidth": 2,
  "trailingComma": "all",
  "overrides": [
    {
      "files": "*.sol",
      "options": {
        "tabWidth": 4
      }
    }
  ]
}
```

File: test/unit/BondingManager.js [hunk 2]
```diff
@@ -1,3807 +1,3868 @@
...
-  solidity
-} from 'ethereum-waffle'
-
-chai.use(solidity)
-const {
-  expect
-} = chai
-const {DelegatorStatus, TranscoderStatus} = constants
+import Fixture from "./helpers/Fixture";
+// import expectRevertWithReason from "../helpers/expectFail";
```
Comment (kyriediculous) [inline test/unit/BondingManager.js]: we can remove this comment

File: test/unit/BondingManager.js [hunk 3]
```diff
@@ -1,3807 +1,3868 @@
...
+import { contractId, functionSig, functionEncodedABI } from "../../utils/helpers";
+import { constants } from "../../utils/constants";
+import math from "../helpers/math";
+import BN from "bn.js";
+// import truffleAssert from "truffle-assertions";
```
Comment (kyriediculous) [inline test/unit/BondingManager.js]: we can remove this comment

File: test/unit/BondingManager.js [hunk 4]
```diff
@@ -1,3807 +1,3868 @@
...
+    const UNBONDING_PERIOD = 2;
+    const MAX_EARNINGS_CLAIMS_ROUNDS = 20;
 
-    const PERC_DIVISOR = 1000000
-    const PERC_MULTIPLIER = PERC_DIVISOR / 100
+    const PERC_DIVISOR = 1000000;
+    const PERC_MULTIPLIER = PERC_DIVISOR / 100;
 
-    let accounts
+    let accounts;
```
Comment (kyriediculous) [inline test/unit/BondingManager.js]: In the other files we call this signers, I think it would be nice if we could stick to that convention.

**Review by kyriediculous — CHANGES_REQUESTED** [review]

File: test/unit/BondingManager.js (hunk 4 above)
Comment (kautukkundan) [reply test/unit/BondingManager.js]: done 👍🏼

**Review by kyriediculous — APPROVED** [review]

**Pull request merged**